GCRCatalogs/catalog_configs/dc2_object_run1.2i_all_columns.yaml

```
@@ -0,0 +1,7 @@
+subclass_name: dc2_object.DC2ObjectCatalog
+base_dir: /global/projecta/projectdirs/lsst/global/in2p3/Run1.2i/object_catalog
+filename_pattern: 'object_tract_*\.hdf5$'
```

Should we be more restrictive? 
```suggestion
filename_pattern: 'object_tract_\d+\.hdf5$'
```